.bandit

```diff
@@ -0,0 +1,2 @@
+[bandit]
+exclude: tests
```

Comment:
Necessary?
Can an existing config file be used if so?

Reply:
Sadly no. It would be great, if this could be moved to the `setup.cfg`, seealso https://github.com/PyCQA/bandit/issues/318

I mean, I also don't need to enable bandit. But it's always a good idea to do some security linting. I can move that to a separate PR tho.